### PR TITLE
Separate CAKE and LP APR

### PR DESCRIPTION
**Pull request by Chef-Cheems**
Show base APR and combined APR separately
Screenshot 2021-07-02 at 10 47 51

### CODE REVIEW

File: src/components/ApyCalculatorModal/index.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ interface ApyCalculatorModalProps {
   onDismiss?: () => void
   tokenPrice: number
   apr: number
+  combinedApr?: string
```
Comment (hachiojidev) [inline src/components/ApyCalculatorModal/index.tsx:11]: A small thing but maybe we should change the name of this. In other places your variable names ending `Apr` are numbers.

Alternatively, make this a number.

Comment (hachiojidev) [reply src/components/ApyCalculatorModal/index.tsx:11]: Maybe add the word `display` like we do in other places.

Comment (Chef-Cheems) [reply src/components/ApyCalculatorModal/index.tsx]: Changed to `displayApr`, what do you think?

**Review by ChefHutch — APPROVED** [review]
👍

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**